src/pages/progress_reports/SubmitProgressReport.tsx

```diff
@@ -3,28 +3,36 @@ import {AxiosError} from 'axios';
 import {ProgressReport} from '../../libs/ajax/ProgressReport';
 import {Notifications} from '../../libs/utils';
 import {ConsentError} from '../../types/responseTypes';
+import {validationFailed} from "src/utils/darFormUtils";
 
 
 interface SubmitProgressReportProps {
   readonly progressReport: object;
   readonly parentReferenceId: string;
   readonly onSuccess: (result: unknown) => void;
   readonly onCancel: (result: unknown) => void;
+  readonly validateForm: () => any;
```

Comment:
See my other comment about this - I think we're putting too much into this component, it should not be enabled unless validation has already happened and complete.